src/main/java/org/folio/marccat/dao/CatalogDAO.java

```diff
@@ -125,14 +137,14 @@ public void updateBibNote(final int amicusNumber, final int noteNumber, final Se
   public void modifyNoteStandard(final CatalogItem item, final Session session) throws HibernateException {
```

Comment:
Does this actually throw HibernateException?  Looks like you catch it.

Reply:
That's right